src/transformers/models/deformable_detr/modeling_deformable_detr.py

```diff
@@ -589,7 +589,9 @@ class DeformableDetrMultiscaleDeformableAttention(nn.Module):
     Multiscale deformable attention as proposed in Deformable DETR.
     """
 
-    def __init__(self, embed_dim: int, num_heads: int, n_levels: int, n_points: int):
+    def __init__(
+        self, embed_dim: int, num_heads: int, n_levels: int, n_points: int, disable_custom_kernels: bool = False
+    ):
```

Comment:
Let's pass the config here (I think you can also remove `n_points` from the args, which seems to always be `config.num_feature_levels`) and the read the flag from the config.

Reply:
Hopefully addressed in 4f3f0416438a9ff531bdc66187d0a7c6f59776cf!